lib/popover.js

```diff
@@ -87,7 +87,7 @@ const PopoverContent = ({ align, children, ...props }) => {
   );
 };
 
-const PopoverContainer = styled.details`
+const PopoverContainer = styled.span`
```

Comment:
This is wrapping all the popover contents (like buttons, headers, paragraphs etc) right? If so it should be a div or maybe `section` - span can only contain phrasing content, not flow content https://developer.mozilla.org/en-US/docs/Web/HTML/Element/span